app/ledger.js

```diff
@@ -695,6 +695,7 @@ var ledgerInfo = {
 
   // the desired contribution (the btc value approximates the amount/currency designation)
   btc: undefined,
+  btcPrice: undefined,
```

Comment:
@willy-b not sure if this is obvious to you, but the BTC price should now be available as a property of `transactions` according to @mrose17. specifically `ledgerInfo.transactions[i].contribution.rates` is the BTC price in the local currency at the time of the transaction (i think).